### PR TITLE
Add two sinhala classification datasets

### PR DESCRIPTION
**Pull request by shreeya-dhakal**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

**Review by dokato — COMMENTED** [review]
I think this looks all good! Thanks for adding first Sinhala datasets. Feel free to add your points following [this guideline](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md).

File: mteb/tasks/Classification/sin/SinhalaNewsClassification.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            year      = {2022},
+            }""",
+        n_samples={"train": 3327},
+        avg_character_length={"train": 148.04},
+    )
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_column(
+            {"comments": "text", "labels": "label"}
+        )
```
Comment (imenelydiaker) [inline mteb/tasks/Classification/sin/SinhalaNewsClassification.py:49]: I'm not sure this works, you're missing an s in `rename_column`, see [docs](https://huggingface.co/docs/datasets/v2.19.0/en/package_reference/main_classes#datasets.Dataset.rename_column)
```suggestion
        self.dataset = self.dataset.rename_columns(
            {"comments": "text", "labels": "label"}
        )
```

Comment (shreeya-dhakal) [reply mteb/tasks/Classification/sin/SinhalaNewsClassification.py:49]: Good catch, I will update it. The original code had two lines that rename the columns so I missed that in this quick update.

**Comment by shreeya-dhakal** [comment]
> I think this looks all good! Thanks for adding first Sinhala datasets. Feel free to add your points following [this guideline](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md).

@dokato @imenelydiaker  since there are two Sinhala datasets the score should be 2+2+4 right 4 for the two datasets and 4 for a new language and I show have two line for contributors, one for each of you?

{"GitHub": "shreeya-dhakal", "New dataset": 8}
{"GitHub": "dokato", "Review PR": 2}
{"GitHub": "imenelydiaker", "Review PR": 2}

**Review by imenelydiaker — APPROVED** [review]

**Comment by imenelydiaker** [comment]
Thanks for your contribution! I'm enabling auto-merge 🙂

**Pull request merged**